server/subscription.go

```diff
@@ -62,7 +72,11 @@ func (s *Server) registerSubscriptionRoutes(g *echo.Group) {
 func (server *Server) loadLicense() (*enterprise.License, error) {
 	license, err := server.LicenseService.LoadLicense()
 	if err != nil {
-		server.l.Warn("Failed to load license", zap.String("error", err.Error()))
+		if common.ErrorCode(err) == common.LicenseInvalid {
+			server.l.Warn("Failed to load valid license", zap.String("error", err.Error()))
+		} else {
```

Comment:
probably we should flip the logic, we check if it's NOT FOUND, then DEBUG, otherwise warn. This is because only NOT FOUND is an expected error.